scripts/azureml-assets/azureml/assets/model/model_utils.py

```diff
@@ -64,6 +64,7 @@ def __init__(self, spec_path, model_config, registry_name, temp_dir, copy_update
         try:
             self._model = load_model(spec_path)
             self._model.description = model_config.description
+            self._model.tags.update(model_config.extra_tags_from_files)
```

Comment:
def resolve_from_file(value):
    if os.path.isfile(value):
        with open(value, 'r') as f:
            content = f.read()
        return content
    else:
        return content

self._model.tags = {i, resolve_from_file(v) for i, v in self._model.tags.items()}

we can use that for all the values, not just tags (out of the scope of these changes)